devlab/tests/test_resource_migration.py

```diff
@@ -120,6 +120,33 @@ def test_migrate_nova_security_groups(self):
                                                 resource_name='security_group',
                                                 parameter='description')
 
+    def test_image_members(self):
+        src_images = self.src_cloud.glanceclient.images.list()
+        dst_images_gen = self.dst_cloud.glanceclient.images.list()
+        dst_images = [x.__dict__ for x in dst_images_gen]
+        filtering_data = self.filtering_utils.filter_images(src_images)
+        src_images = filtering_data[0]
+
+        src_members = []
+        dst_members = []
+        for image in src_images:
+            member = self.src_cloud.glanceclient.image_members.list(image['id'])
+            if len(member) > 0:
+                members_list = []
+                for mem in member:
+                    formatted_mem = mem.__dict__['_info']
+                    members_list.append(formatted_mem)
+                src_members.append({image['id']: members_list})
+        for image in dst_images:
+            member = self.dst_cloud.glanceclient.image_members.list(image['id'])
+            if len(member) > 0:
+                members_list = []
+                for mem in member:
+                    formatted_mem = mem.__dict__['_info']
+                    members_list.append(formatted_mem)
+                dst_members.append({image['id']: members_list})
```

Comment:
The only difference between first and second loop is `src_` and `dst_` prefix. Please refactor